qemu/tests/timerdevice_clock_drift_with_sleep.py

```diff
@@ -28,12 +28,14 @@ def run(test, params, env):
     """
     def verify_elapsed_time():
         usleep_cmd = r'echo "for n in \$(seq 1000);'
-        usleep_cmd += ' do usleep 10000; done"'' > /tmp/usleep.sh'
+        usleep_cmd += ' do sleep 0.01; done"'' > /tmp/usleep.sh'
         session.cmd(usleep_cmd)
 
         get_time_cmd = 'for (( i=0; i<$(grep "processor" /proc/cpuinfo'
         get_time_cmd += ' | wc -l); i+=1 )); do /usr/bin/time -f"%e"'
         get_time_cmd += ' taskset -c $i sh /tmp/usleep.sh; done'
+        guest_cpu = session.cmd_output("grep 'processor' /proc/cpuinfo | wc -l")
+        timeout = int(guest_cpu) * 12
```

Comment:
Better use a different variants name, to distinguish with the login timeout, or directly assign it to `timeout=int(guest_cpu) * 12`) is also okay here.